.github/workflows/linter.yml

```diff
@@ -3,9 +3,18 @@ name: Lint Code Base
 on:
   push:
     branches: [ master ]
+    paths-ignore:
+      - README.md
+      - LICENSE
+      - .github/workflows/
```

Comment:
```suggestion
      - .github/workflows/*
```

Reply:
It wouldn't work otherwise.

Get a better understanding: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet